### DISCUSSION TITLE
How to link fonts for a portable version

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1584] [Q&A]
What's the right way to add a path to a font in a theme?
At the moment, it looks like this:

```
src: url('./fonts/IBM-Plex-Mono-woff2/IBMPlexMono-Regular.woff2') format('woff2');
```

But then I get this

```
Failed to load resource: the server responded with a status of 404 (Not Found)
http://127.0.0.1:37840/api/notes/download/fonts/IBM-Plex-Mono-woff2/IBMPlexMono-Regular.woff2\
```

**Comment by zadam** [comment]
Check how this is done in the demo document in the custom "Steel Blue" theme.

Part of the solution is to expose the fonts via HTTP using http://github.com/zadam/trilium/wiki/Custom-request-handler
